### DISCUSSION TITLE
CHOIX DE L'ORDRE DU MODELE

### DISCUSSION THREAD

**Discussion by KLOSTRO** [FrSkyRC/ETHOS-Feedback-Community#4855] [General]
Il a été annoncé qu'il serait possible dans la version 1.6.0 que l'on pourrait organiser l'ordre de nos modèles autre qu'en alphabétique comme actuellement ?
Cela sera t il comme effectif avec cette version ?

Merci

**Comment by bsongis-frsky** [comment]
Ethos 1.6.0 est sortie beaucoup plus tôt que prévu pour être stable en même temps que la RF 3.0
Tout ce qui a été prévu dans cette version sera repoussé à la branche 1.7

**Comment by KLOSTRO** [comment]
Bonjour Bertrand
OK merci pour ta réponse, je suis impatient de pouvoir utiliser la version 1.7.00xx

Merci pour ta réponse
